get-build-information.groovy

```diff
@@ -0,0 +1,30 @@
+//This script allows you to get build information for selected jobs
+/*
+ You can also add closures to filter the jobs or the builds...
+ .find{job->job.name.contains('dsl')}
+ .find{ [whatever]}
+
+*/
+Jenkins.instance.getAllItems(Job).each{
+
+  def jobBuilds=it.getBuilds()
+
+	//for each of such jobs we can get all the builds (or you can limit the number at your convenience)
+    jobBuilds.each { build ->
+      def runningSince = groovy.time.TimeCategory.minus( new Date(), build.getTime() )
+      def currentStatus = build.getBuildStatusSummary().message
+      def cause = build.getCauses()[0] //we keep the first cause
```

Comment:
Uh, what if `causes` is empty? You will get an `ArrayIndexOutOfBoundsException`.